test_environment.yml

```diff
@@ -0,0 +1,18 @@
+name: Qutip-Test-Env
```

Comment:
```suggestion
name: test-environment
```

Reply:
The capitalized Qutip-Test-Env just looks odd to me and makes it sound like this is an environment for testing QuTiP itself. :)